### ISSUE TITLE
UI improvement

### ISSUE THREAD

**Issue by mlamond** [ThePacielloGroup/CCAe#111]
**Is your feature request related to a problem? Please describe.**
I think the UI is a little visually cluttered and distracting compared to the previous version. The most important pieces of information (the pass/fail and the contrast ratio) get lost among the other page elements and are disconnected from each other--contrast ratio all the way at the top of the screen, pass/fail at the bottom. 

**Describe the solution you'd like**
I would prefer a cleaner and simpler design, where the resulting contrast ratio and the pass/fail status take visual precedence. The selected colors can also take up less visual space than they do in this new version. The text field to enter a color should look like a text input box. (Right now, it sort of blends into the background and does not appear clicakble until you click on it.) The accordions and other buttons could be much more visually subdued.

**Describe alternatives you've considered**
I continue to use the previous version for the majority of my analyses. I only use this new version to cross-check colors that I think might be cutting it close. The previous version may not have as modern a look, but I prefer the simpler interface that allows the most important information to stand out the most.

**Additional context**
